### PR TITLE
Move and separate changelog

### PR DESCRIPTION
**Pull request by albi3ro**
Our old changelog was over 5,000 lines long, and getting longer all the time.  This made it difficult to navigate and manage.

This PR breaks the changelog into separate files per release in the `doc/releases` folder.

We also request one name per line in the authors' list.

### CODE REVIEW

**Review by josh146 — APPROVED** [review]
Looks good from my end @albi3ro! Thanks for splitting it up.

I'm curious how we get rid of the Sphinx warnings 🤔 The reason they didn't appear before was that the old `.CHANGELOG.md` file wasn't in the doc folder. Sphinx has a really annoying habit of assuming that all text files in `doc/` should be in the TOC.

File: doc/releases/changelog-0.1.0.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Release 0.1.0
```
[the remark this answers is not in the document]
Comment (josh146) [reply doc/releases/changelog-0.1.0.md:3]: orphan seems to work!

**Comment by antalszava** [comment]
This is looking really cool! :heart_eyes: The changelog will be much cleaner with this addition.

Are there maybe any references to the changelog/how to add an entry to the changelog in the docs that need an update? It might be that contributors will need some guidance as to which file to edit exactly.

**Comment by josh146** [comment]
> Are there maybe any references to the changelog/how to add an entry to the changelog in the docs that need an update? It might be that contributors will need some guidance as to which file to edit exactly.

I would imagine that contributors only need to ever add to `changelog-dev.md`?

The only exception would be during a feature freeze, where we may potentially point them to edit `changelog-0.18.md` instead. But perhaps it makes more sense to do this during code review, rather than add discussions on feature freezes to the developer docs.

**Review by antalszava — APPROVED** [review]
Looks good to me! :100:

**Pull request merged**